src/test/skript/tests/syntaxes/conditions/CondIsDivisibleBy.sk

```diff
@@ -0,0 +1,6 @@
+test "is divisible":
+	assert 5 can be evenly divided by 5 with "5 can be divided by 5!"
+	assert 5 can not be evenly divided by 10 with "5 cannot be divided by 10!"
+	assert 5 isn't evenly divisible by 0 with "nothing can be divided by 0!"
+	assert 1964903306 is evenly divisible by 982451653 with "1964903306 is divisible by 982451653!"
+	assert {_none} is evenly divisible by 10 to fail with "you cannot divide by <none>!"
```

Comment:
Let's add some tests where the first input is a list (since the syntax supports that)